project-management/1-1-meetings/Anne-Lee-Steele/2024-checkins-malvika.md

```diff
@@ -0,0 +1,185 @@
+## 1:1 meetings
+
+# 23-04-2024
+- Preparation for community forum
+- Community health - presenting tomorrow at RCM team meeting
```

Comment:
Add three grand changes from Turing: what can we learn about health (body), environment (sustainability and well-being), & defense (protection)